src/main/java/org/tndata/android/compass/adapter/CategoryFragmentAdapter.java

```diff
@@ -84,6 +89,7 @@ public void toggleCard() {
     private Category mCategory;
     private List<Goal> mItems;
     private CategoryFragmentAdapterInterface mCallback;
+    private static final String TAG = "CategoryFragmentAdapter";
 
```

Comment:
Be careful here in the future, an error occurs when the Log tag is too long. I think the max is either 23 or 25, and this is getting close..